.github/workflows/version-bump.yml

```diff
@@ -0,0 +1,50 @@
+name: Version bump
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version (major.minor.patch)'
+        required: true
+      abi-version:
+        description: 'ABI version (major.minor.patch)'
+        required: true
+
+env:
+  VERSION: ${{ github.event.inputs.version }}
+  ABI_VERSION: ${{ github.event.inputs.abi-version }}
+  GITHUB_TOKEN: ${{ github.token }}
+
+jobs:
+  bump-version:
+    name: "Bump version"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Criterion
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure git
+        run: |
+          git config user.name "github-actions"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Does this have any significance or should we just pick some Acme robot email?

Reply:
That weird string seems to be the email address of @github-actions. I've added it so that commits from the bot appear correctly on the GitHub UI.